app/controllers/languages_controller.rb

```diff
@@ -31,15 +30,32 @@ def edit
   def update
     @language = Language.find_by(short: params[:id])
     authorize @language
+    
+    unless User.current_user.roles.compact.flatten.include?("superadmin") || User.current_user.roles.compact.flatten.include?("translation")
```

Comment:
I would leave out this separate check.

Reply:
Can there be a case where a user is both a translation admin and a support admin? If so I feel like the check is needed.